src/main.ts

```diff
@@ -1,17 +1,27 @@
 import { NestFactory } from '@nestjs/core'
 import { ConfigService } from '@nestjs/config'
 import { DocumentBuilder, SwaggerModule } from '@nestjs/swagger'
+import { env } from 'process'
+import session from 'express-session'
 import { AppModule } from './modules/app/app.module'
 import { configurePipes } from './bootstrap/pipes'
-import { env } from 'process'
 
 async function bootstrap(): Promise<void> {
   const app = await NestFactory.create(AppModule)
   const configService = app.get(ConfigService)
 
+  app.setGlobalPrefix('api')
   app.enableCors()
   configurePipes(app)
 
+  app.use(
+    session({
```

Comment:
what are you going to use in production instead of it?
`The default server-side session storage is purposely not designed for a production environment. It will leak memory under most conditions, does not scale past a single process, and is meant for debugging and developing. Read more in the official repository https://github.com/expressjs/session`